scripts/header.js

```diff
@@ -8,4 +8,17 @@
             Add name of state park
 
         Return HTML for header
-*/
\ No newline at end of file
+*/
+const parkName = "Cider Falls Park"
+const logoURL = "URL"
+
+
+
+const getHeader = () => {
+    const headerHTML = `
+        <header>
+            <img src="${logoURL}" alt="Park Logo">
+            <h1>${parkName} State Park </h1>
```

Comment:
Is this supposed to make it say "Cider Falls Park State Park"?

Reply:
its going to pull it from the parkName on line 12